aiida/tools/visualization/graph.py

```diff
@@ -279,25 +310,13 @@ def _add_graphviz_node(
     types, and also color the nodes for successful/failed processes
     """
     # pylint: disable=too-many-arguments
-    node_style = {}
-    if isinstance(node, orm.Data):
-        node_style = node_style_func(node)
-        label = ['{} ({})'.format(node.__class__.__name__, get_node_id_label(node, id_type))]
-
-    elif isinstance(node, orm.ProcessNode):
-        node_style = node_style_func(node)
-
-        label = [
-            '{} ({})'.format(
-                node.__class__.__name__ if node.process_label is None else node.process_label,
-                get_node_id_label(node, id_type)
-            )
-        ]
+    node_style = node_style_func(node)
+    label = [_get_node_label_text(node, id_type)]
 
     if include_sublabels:
-        sublabel = node_sublabel_func(node)
-        if sublabel:
-            label.append(sublabel)
+        sublabel_text = node_sublabel_func(node)
+        if sublabel_text:
+            label.append(sublabel_text)
```

Comment:
I use 'text' as suffix to distinguish the type of these variable from the type of `label` where `label` is a `list` here, so I think it might be more readable to name the inside text of label as label_text and sublabel_text. And that is why I named the isolated function `_get_node_label_text` rather than `_get_node_label`.

Reply:
Ah, I see. This is actually a good reason. I would ideally prefer to clarify this the other way around (I would expect the `sublabel` to be a string/text, so I would actually preferr that the other one is called `label_list`), but it is ok if you want to leave it like this.

Reply:
Good idea! I'll adopt your advice.